ui/imports/Themes/DarkTheme.qml

```diff
@@ -49,6 +49,7 @@ Theme {
     property color pillButtonTextColor: almostBlack
     property color chatReplyCurrentUser: darkerGrey
     property color topBarChatInfoColor: darkerGrey
+    property color codeBackground: "#2E386B"
```

Comment:
FYI, it's not the end of the world, but  you should add it to `Theme` as well with an empty value just so the interface knows the value and then Qt Creator can also correctly autocomplete the values

Reply:
Added... not sure if I added it to the right `Theme.qml` though.